src/lib/Utils.js

```diff
@@ -65,4 +67,10 @@ export const fromUiDeciamlPointFormat = (num, decimalPoint) => {
     const x = toBN(toWei(num))
     return x.div(factor)
 }
-  
\ No newline at end of file
+  
+export const gasToEth = async (gas, web3) => {
+    const gasPrice = await web3.eth.getGasPrice()
+    const cost = gas * gasPrice
```

Comment:
do we need big num math here?

Reply:
gas is in Wei
and gas price is in Gwei (8 numbers after the decimal)
these are pretty small numbers

and I compared them to the calculation meta mask is making...